packages/shared/components/popups/MintNftFormPopup.svelte

```diff
@@ -94,7 +98,7 @@
             return
         }
 
-        const isKeysValid = Object.keys(royalties).every((key) => !validateBech32Address($networkHrp, key))
+        const isKeysValid = Object.keys(royalties).every((key) => validateBech32Address($networkHrp, key))
```

Comment:
This logic is duplicated

Reply:
I created a new issue for this: https://github.com/iotaledger/firefly/issues/5289